tests/test_samples.py

```diff
@@ -1067,7 +1070,7 @@ def test_logzero_mask_likelihood_level():
 
     ns1 = read_chains('./tests/example_data/pc')
     ns1.logL = np.where(mask, ns1.logL, -1e30)
-    ns1 = merge_nested_samples((ns1[ns1.logL > ns1.logL_birth],))
+    ns1 = merge_nested_samples((ns1.loc[ns1.logL > ns1.logL_birth],))
```

Comment:
```python
mask = ns1.logL.to_numpy() > ns1.logL_birth.to_numpy()
ns1 = merge_nested_samples((ns1[mask],))
```

Reply:
`mask` is already used in this test, so I'll give it a different name

Reply:
mask2 is the limit of my creativity for now

Reply:
I think overwriting `mask` would be fine, wouldn't it?

Reply:
as you wish